src/components/BulkManagementHistoryView/index.jsx

```diff
@@ -12,8 +12,8 @@ import HistoryTable from './HistoryTable';
  */
 export const BulkManagementHistoryView = () => (
   <div className="bulk-management-history-view">
-    <h4><FormattedMessage {...messages.heading} /></h4>
-    <p className="help-text">
+    <h4 className="font-weight-bold mt-0"><FormattedMessage {...messages.heading} /></h4>
+    <p className="pb-5 w-75">
```

Comment:
w-75 sets the width, not the max width, and pb-5 is for padding. I think we need the margin here. 
I think mb-5 would be closer, but we can undo this for since it isn't an exact equivalent.

Reply:
You are right, reverting this change.